### PR TITLE
fixing visibility of terms in footer

### LINKED ISSUE #4005: "Terms" is not accessible in some resolutions
#### Excepted Interface
image

#### Unaccesible 'Term' in Desktop view 

image

#### Unaccesible 'Terms' in Mobile view  

image

### PR DESCRIPTION
**Pull request by s2sharpit**
**Description** - The visibility and accessibility of 'Terms' in the footer of [Layer5.io](https://www.layer5.io/) are blocked by navigation arrows in some resolutions. Fixed this, by adjusting some elements in the footer for different resolutions.

This PR fixes #4005 
Signed off by: Tushar Saini

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 02ebebb0f1df158e808247be4d4300a2dc486a16 at: https://6420068c7a2d317e8cfe083d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit ba3c891acfb0220fae9be961ea81eca4dfc4bff5 at: https://64200c397164c00348f56762--layer5.netlify.app

**Comment by s2sharpit** [comment]
Hey @Chadha93, I have signed off the commit. Have a check.

File: src/sections/General/Footer/footer.style.js [hunk 1]
```diff
@@ -234,23 +236,35 @@ const FooterWrapper = styled.section`
...
+			// 	li {
+			// 		padding: 0.125rem
+			// 	}
+			// }
+
+			// li + li {
+			// 	margin-left: 0 !important;
+			// 	text-align: center;
+			// }
 		}
```
Comment (Nikhil-Ladha) [inline src/sections/General/Footer/footer.style.js:268]: Instead of commenting, remove it

File: src/sections/General/Footer/footer.style.js [hunk 2]
```diff
@@ -234,23 +236,35 @@ const FooterWrapper = styled.section`
...
+		padding-right: 0;
+
 		.footer-bottom{
-			padding: 0 1rem;
+			flex-direction: column;
+			align-items: center;
+			padding: 0 1.2rem;
+
+			.copyright-text {
+				// order: 2;
```
Comment (Nikhil-Ladha) [inline src/sections/General/Footer/footer.style.js:248]: Remove the commented code

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit c3eb92b6f75c4adc91492071ecdc64fdab87d9b8 at: https://64202cacf7a788203f26a523--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]
Looks good.
Nice work @s2sharpit 👌🏼

**Pull request merged**